RaftMerge/RaftMerge.tla

```diff
@@ -230,6 +235,7 @@ InternalRequest(i, r, log) ==
 \* Send merge request to the leader of Region B.
 ProposeMergeRequest(i) ==
   /\ raft[i][RegionB].is_leader
+  /\ region[i][RegionB] = RegionNormal
```

Comment:
I add this line just to keep symmetry with L252. In fact it should not affect anything, because merge request will be proposed only once, and once applied, it will turn to `RegionMerging` state.

Reply:
lol